python/py-mac-alias/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
```

Comment:
please use PyPI as the source , preferred for Python packages, instead of GitHub

Reply:
I took a random `py-*` Portfile and adapted it for this module. Can you point me to a better example to adapt?

Also, why is PyPI preferred as a source? Isn’t the GitHub repository also the source for PyPI?

Reply:
PyPI is the source MacPorts prefers if possible, which is also the repository used by, for example, pip.

Reply:
I didn't know about `upt`. I just tried it, but it didn't result in a working port, so I think I'll just keep this MR as-is. I think I've modified the port to use PyPI now.

Reply:
well, you'll have to do some manual work but the majority of the field should be automatically set correctly.  If there are things obviously wrong with the output from `upt`, then please let us know (here or in the repository).